Riot/ViewController/Widgets/WidgetPickerViewController.m

```diff
@@ -53,15 +53,9 @@ - (void)showInViewController:(MXKViewController *)mxkViewController
 
     MXRoom *room = [mxSession roomWithRoomId:roomId];
 
-    NSArray<Widget*> *roomWidgets = [[WidgetManager sharedManager] widgetsNotOfTypes:@[kWidgetTypeJitsi]
+    NSArray<Widget*> *widgets = [[WidgetManager sharedManager] widgetsNotOfTypes:@[kWidgetTypeJitsi]
                                                                           inRoom:room];
 
-    NSArray<Widget*> *userWidgets = [[WidgetManager sharedManager] userWidgets:room.mxSession];
```

Comment:
because we want to list only room widgets in this list.
I temporarily used that list to have access to the sticker picker but now we have UI to open it.